src/index.js

```diff
@@ -203,10 +193,10 @@ const externalLinkHandler = (e, url) => {
  * @param {Event} e
  * @param {String} url
  */
-const aboutLinkHandler = (e, url) => {
-  e.preventDefault();
-  shell.openExternal(url);
-};
+// const aboutLinkHandler = (e, url) => {
```

Comment:
I forgot to say. Besides that I had to download more plugins, I also needed to comment it out cause of eslint complaints

Reply:
I forgot to add the plugins to the package-json after updating ESLint configuration, so that's on me! 👀

As for the eslint complains, what are they?

Reply:
They say that external link handler isn't in use

Reply:
![image](https://user-images.githubusercontent.com/57839948/107553781-657f9c00-6bd5-11eb-8931-7b95eafaf5b6.png)

Reply:
I tried replying via email to your previous reply but that apparently failed. 

> I'll try to reproduce it on my machine and get back to you. In the mean time you could work on it and ignore eslint for now. You may also use `// eslint-disable-next-line no-unused-vars`. I'm not sure I wrote that 100% correctly so better try it or check the docs for it.

Reply:
I found the issue! I forgot to add it to aboutWindow 😆

Reply:
Sorry for that, will fix

Reply:
Just as I was cloning your repo 😂😂

Reply:
Also, that's `@param {import("electron/main").Event} e`. `Event` is the `lib.dom.d.ts` type, not Electron's.

Reply:
what type of event is it? and how does it matter?

Reply:
It matters for code completion and readability. `Event` is the DOM event, while the syntax I provided above yields Electron's `Event`. The exact type is `Electron.NewWindowWebContentsEvent`. 

I just noticed that this type was already set, so you're free to correct it or leave it as is provided it's not part of this PR's responsibility.